src/main/scala/bookbot/models/TitleAndAuthor.scala

```diff
@@ -0,0 +1,26 @@
+package bookbot.models
+
+import bookbot.models.errors.TransformError
+import bookbot.models.errors.Error
+import Error.ErrorOr
+
+case class TitleAndAuthorParseError(message: String) extends Error {
+  def getMessage = message
+}
+
+object TitleAndAuthor {
+  type TitleAndAuthor = (String, String)
+
+  def from(input: String): Either[Error, TitleAndAuthor] = {
```

Comment:
As of now this is only returning one `Error`, so the return type could be more specific here: `Either[TitleAndAuthorParseError, TitleAndAuthor]'